pom.xml

```diff
@@ -322,6 +322,13 @@
                         </systemProperties>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <configuration>
+                        <argLine>-Duser.language=en -Duser.region=US</argLine>
```

Comment:
@krzyk this are system properties must be set on the command line of the forked VM, and cannot be set after the VM has been started.